Source/Extensions/Blazorise.Charts.Streaming/wwwroot/charts.streaming.js

```diff
@@ -96,4 +94,47 @@ function getStreamingOptions(dotNetAdapter, vertical, chartOptions, streamOption
     }
 
     return horizontalScalesOptions;
+}
+
+function getRealtimeOptions(chart) {
+    if (chart && chart.options && chart.options.scales) {
+        const scales = chart.options.scales;
+
+        if (scales.x && scales.x.realtime) {
+            return scales.x.realtime;
+        }
+        else if (scales.x && scales.x.realtime) {
+            return scales.x.realtime;
```

Comment:
The else logic looks the same, is it not? Was it supposed to fallback to **y** or something?

Reply:
Yep, it should be `y`.